scripts/cfggen/main.go

```diff
@@ -70,19 +72,19 @@ func main() {
 
 	errLogger := level.Error(log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr)))
 	if _, err := app.Parse(os.Args[1:]); err != nil {
-		errLogger.Log("err", err)
+		_ = errLogger.Log("err", err)
```

Comment:
It's not necessary to check for `err` [here](https://github.com/thanos-io/objstore/blob/main/.errcheck_excludes.txt#L1) in Log() calls.

Reply:
This was mainly a thing that my environment complained about. I saw it wasnt needed after all, but thought it might be a little cleaner anyway.